kratos/geometries/tetrahedra_3d_10.h

```diff
@@ -621,6 +621,20 @@ template<class TPointType> class Tetrahedra3D10 : public Geometry<TPointType>
         return faces;
     }
 
+    /** This method calculates and returns the average edge length of the geometry
+     *
+     * @return double value with the average edge length
+     *
+     */
+    double AverageEdgeLength() const override {
+        double average_edge_length = 0.0;
+        GeometriesArrayType edges = this->GenerateEdges();
+        for (auto& edge : edges) {
+            average_edge_length += edge.Length();
+        }
+        return average_edge_length/6.0;
```

Comment:
Technically it will by faster to multiply by 0.16666666666666666666666

Reply:
the compiler converts divisions with literals to multiplications if it's safe to do so (it's not in this case, because 1/6 is not exactly representable with floating point nums)

Reply:
done!